main.cpp

```diff
@@ -83,26 +85,34 @@ struct Computer
 
     bool connectToInternet(bool paidISP);
     bool save(std::string fileName, bool clickSaveButton);
-    bool runApp(bool doubleClick);
+    void runApp(std::string something);
 };
 
+Computer::Computer()
+{
+    std::cout << "Computer being constructed!" << std::endl; 
+}
+
 bool Computer::connectToInternet(bool paidISP = true)
 {
     return paidISP ? true : false;
 }
 
-bool Computer::save(std::string fileName, bool clickSaveButton)
+bool Computer::save(std::string fileName = "Sweet Cat Video", bool clickSaveButton = true)
 {
     return fileName != "" && clickSaveButton ? true : false;
 }
 
-bool Computer::runApp(bool doubleClick)
+void Computer::runApp(std::string text = "App is running")
 {
-    return doubleClick ? true : false;
+    std::cout << text << std::endl;
 }
```

Comment:
Default arguments should be defined where the function is declared, not where the function is defined. 
- [ ] revise